indico/modules/events/abstracts/controllers/management.py

```diff
@@ -57,6 +57,18 @@ def _process(self):
         return RH._process(self)
 
 
+class RHManageAbstractBase(AbstractMixin, RHManageAbstractsBase):
+    """Base class for all abstract management RHs"""
+
+    def _checkParams(self, params):
+        RHManageAbstractsBase._checkParams(self, params)
+        AbstractMixin._checkParams(self)
+
+    def _checkProtection(self):
+        RHManageAbstractsBase._checkProtection(self)
+        AbstractMixin._checkProtection(self)
```

Comment:
This only checks for read access, not for management access. We need a `can_manage` method that checks whether the guy is an event manager or `can_review()` the abstract.